src/ElasticSearchIndexer.php

```diff
@@ -214,4 +225,15 @@ private function extendBatch(string $action, string $indexName, IndexDocument $d
 
         return $params;
     }
+
+    private function parseErrors(array $clientResponse): string
+    {
+        $errors = '';
+        
+        foreach ($clientResponse['items'] as $response) {
+            $response = reset($response);
+            $errors .= $response['error']['reason'] . '; ';
```

Comment:
as we discussed , let's add a identifier to the processed data so we are able to identify the failing items  
also let's check what items are failing